voyager-transitions/src/commonMain/kotlin/cafe/adriel/voyager/transitions/ScreenTransition.kt

```diff
@@ -39,6 +39,14 @@ public interface ScreenTransition {
      * @return ExitTransition or null when it should not define a transition for this screen.
      */
     public fun exit(lastEvent: StackEvent): ExitTransition? = null
+
+    /**
+     * Defines the z-index for the Screen.
+     *
+     * @param lastEvent - lastEvent in the navigation stack.
+     * @return Float value for the z-index.
+     */
+    public fun zIndex(lastEvent: StackEvent): Float = 0f
```

Comment:
Can we define zIndex as null? Mostly because the other functions (exit and enter) when null it use the default behavior defined in the `ScreenTransition`, but, this is not the case for `zIndex` here.